srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -27,23 +183,34 @@
     // if not.
     private RiskModel importedModel = null;
     
-    protected EditRiskModel(final RiskModel rm)
+    protected EditRiskModel(final RiskModel rm, final List<Specialty> applSpecialty)
     {
         fRiskModel = rm;
         modelName = rm.getDisplayName();
```

Comment:
I'm still getting used to this. What does the 'f' stand for?  I've only seen this convention where the first char indicates the type, like float.  I never really liked that though.